storm-server/src/main/java/org/apache/storm/nimbus/AssignmentDistributionService.java

```diff
@@ -186,7 +186,6 @@ private Integer nextQueueId() {
      * Get an assignments from the target queue with the specific index.
      * @param queueIndex index of the queue
      * @return an {@link NodeAssignments}
-     * @throws InterruptedException
```

Comment:
Why was this removed?

Reply:
Answered [elsewhere](https://github.com/apache/storm/pull/3050#discussion_r312710620)